Gemfile

```diff
@@ -5,4 +5,5 @@ group :development do
   gem 'rspec', '~> 3.1'
   gem 'ruby-prof', '~> 0.15'
   gem 'simplecov', '~> 0.9'
+  gem 'multi_json', '~> 1.4'
```

Comment:
Is there a reason why we can't use 1.11.0? Feel free to upgrade other gems if they're preventing us.